File: experiment_jp_test/mutants/for__0__COUNT_WAYS_BUILD_STREET_GIVEN_CONSTRAINTS.java
static long countWays ( int n ) { long dp [ ] [ ] = new long [ 2 ] [ n + 1 ] ; dp [ 0 ] [ 1 ] = 1 ; dp [ 1 ] [ 1 ] = 2 ; return dp [ 0 ] [ n ] + dp [ 1 ] [ n ] ; }

File: experiment/java-python/test/j2pyrawpy/for__0__COUNT_WAYS_BUILD_STREET_GIVEN_CONSTRAINTS.py
#!/usr/bin/env python
""" generated source for module for__0__COUNT_WAYS_BUILD_STREET_GIVEN_CONSTRAINTS """
class X(object):
    """ generated source for class X """
    @classmethod
    def countWays(cls, n):
        """ generated source for method countWays """
        dp = [None]*2
        dp[0][1] = 1
        dp[1][1] = 2
        return dp[0][n] + dp[1][n]

